src/sparseml/core/state.py

```diff
@@ -186,6 +195,13 @@ def update(
             if batches_per_step is not None:
                 self.start_event.batches_per_step = batches_per_step
 
+        loggers = loggers or []
+        if isinstance(loggers, List):
+            loggers = LoggerManager(loggers)
```

Comment:
what happens in `[]` scenario? is it safe to have loggers not be a `LoggerManager` instance?